tools/pipelines/build-client.yml

```diff
@@ -139,3 +139,25 @@ extends:
     - syncpack:versions
     - check:versions
     - prettier
+    additionalBuildSteps:
+      - task: Bash@3
+        displayName: Inject devtools telemetry logger token
+        inputs:
+          script: |
+            echo Generating .env
+            echo "DEVTOOLS_TELEMETRY_TOKEN=$(devtools-telemetry-key)" >> ./packages/tools/devtools/devtools-browser-extension/.env
+            cat ./packages/tools/devtools/devtools-browser-extension/.env
```

Comment:
This will print the secret in the logs, right? Isn't the goal to avoid leaking the token?

Reply:
The goal is to avoid leaking it in source control in particular. Having it in our internal logs is not the end of the world (it ends up being public in the extension bundle), but I don't think it provides a huge amount of value so I'm ok removing the `cat`.

Reply:
Removed